releaseNotes/master.md

```diff
@@ -23,6 +23,9 @@ ${RES_VER_DATE}
   - Fixes
       - **simple title**: brief description
 
+## Depreciation Notice
+  - **lastHubspotSyncAt field removed from accounts GET API response**: Accounts GET API will no longer return `lastHubspotSyncAt` field in the response body. So we suggest removing this field if you are using it anywhere in your application built using Shippable API.
```

Comment:
The second line is unnecessary. I think it's clear that there is no alternative if the field is not being returned.